package/src/frontend/media/MediaPool.js

```diff
@@ -44,7 +44,8 @@ export class MediaPool {
 
     player = this.unAllocatedPlayers[playerType].pop();
     if (player) {
-      
+
+      player.el().children[0].setAttribute('alt', altText); //add alt texts;
```

Comment:
**suggestion**: Use `getMediaElement`

Two lines down we also change attributes on the media element. By using `getMediaElement`, we get around duplicating the knowledge that VideoJS wraps the media element in a div. The comment does not add any value.

Reply:
**suggestion**: Add test

This is covered indirectly through the React component tests. Those components belong to a different package (`pageflow-scrolled`), though. We could also add a test in the `pageflow` package. Similar to [the one for loop](https://github.com/codevise/pageflow/blob/93180fde9abe03ae24b7aff37969083a44bd3b96/package/spec/frontend/media/media_spec.js#L44)

Reply:
Done